OData/src/System.Web.OData/OData/Routing/Conventions/AttributeRoutingConvention.cs

```diff
@@ -53,8 +53,14 @@ public AttributeRoutingConvention(IEdmModel model, HttpConfiguration configurati
                 throw Error.ArgumentNull("configuration");
             }
 
+            DefaultODataPathHandler odataPathHanlder = pathTemplateHandler as DefaultODataPathHandler;
```

Comment:
so we also need to pass configuration there, but other code path doesn't have such configuration.